samples/SmokeTest/SmokeTest/KeyVaultTest.cs

```diff
@@ -7,7 +11,7 @@ namespace SmokeTest
 {
     class KeyVaultTest
     {
-        private const string SecretName = "SmokeTestSecret";
+        private static string SecretName = "SmokeTestSecret-" + Guid.NewGuid();
```

Comment:
Use string interpolation instead of concatenation for better readability

```
private static string SecretName = $"SmokeTestSecret-{Guid.NewGuid()}";
```

More info: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated